docs/INSTALL.md

```diff
@@ -607,6 +607,9 @@ Set up your developer account and add certificates:
 - Click "+" and choose "Apple Development".
 - You may also need to register your Mac in your Apple Developer account.
 
+If you want to run Organic Maps on a real device, you have to be part of the Organic Maps development team. You can't run the app under
+your own team because you don't have the CarPlay entitlement. Please contact us to request access to the developemnt team.
```

Comment:
Well, you can setup your own CarPlay entitlement...

Reply:
I contacted Apple and tried to get an entitlement. They've never responded to my request and after one month I don't think there will be any response. I just think future developer should be aware of that to safe them some time.

Reply:
At least its worth mentioning in the docs.

Reply:
I added the entitlement part. Let me know if I should changed the wording.